src/gtimelog/tests/core/test_exports.py

```diff
@@ -0,0 +1,154 @@
+import doctest
+import unittest
+import sys
+from datetime import datetime, time
+from io import StringIO
+
+import freezegun
+from unittest import mock
```

Comment:
unittest is a stdlib module and this import belongs in the 1st section.

(isort will take care of everything once I configure it)

Reply:
Applied